pkgs/tools/networking/gmrender-resurrect/default.nix

```diff
@@ -19,6 +19,14 @@ in
       sha256 = "14i5jrry6qiap5l2x2jqj7arymllajl3wgnk29ccvr8d45zp4jn1";
     };
 
+    patches = [
+      (fetchpatch {
```

Comment:
Usually it would be nice to add a short one-line comment that explains why the patch is necessary (`# libupnp 1.14 compatiblility`) and maybe links the upstream PR. That's not important enough to block this PR here, since the purpose is obvious enough. Still, for next time :)

Reply:
sure, thanks for the heads up ;)